mmdeploy/backend/vacc/backend_manager.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import logging
+import sys
+from typing import Any, Callable, Optional, Sequence
+
+from mmdeploy.utils import get_common_config, get_model_inputs, get_root_logger
+from ..base import BACKEND_MANAGERS, BaseBackendManager
+
+
+@BACKEND_MANAGERS.register('vacc')
+class VACCManager(BaseBackendManager):
+
+    @classmethod
+    def build_wrapper(cls,
+                      backend_files: Sequence[str],
```

Comment:
`backend_files` are supposed to be used for passing the backend model's files, for example the `model_info_json` and `vdsp_params_info_json` 

```python
model_info_json = common_cfg['model_info']
vdsp_params_info_json = common_cfg['vdsp_params_info']
```

Reply:
Yes. But vacc python forward sdk only support forward model by two json files now, so we will update this way next version by backend model's files.